types/types.go

```diff
@@ -26,6 +27,23 @@ type App struct {
 	settings        settings.Settings
 	metadataContent []byte
 	metadata        metadata.AppMetadata
+	attachments     []Attachment
+}
+
+// Attachment is a summary of an attachment (attached file) stored in the app definition
+type Attachment struct {
+	filePath string
+	fileSize int64
+}
+
+// FilePath returns the file path
+func (f *Attachment) FilePath() string {
```

Comment:
nit: Since the context is that this is a file, could we call this `Path`? @silvin-lubecki